aries-site/src/data/structure.js

```diff
@@ -115,8 +118,28 @@ export const structure = [
     pages: [],
     pageLayout: 'plain',
   },
+  {
+    name: 'Design tokens',
+    color: 'purple',
+    description:
+      'Design tokens represent design decisions such as color, typography, spacing, and motion in a consistent, reusable, and tech-agnostic format.',
+    icon: (_, color) => <Cube color={color} />,
```

Comment:
Can we add an `role="decorative"` or `aria-hidden`?

Reply:
Done. Added for the other icons as well.